packages/components-js/tests/unit/fixtures/stats.json

```diff
@@ -7,12 +7,12 @@
   {
     "chunkShortName": "banner",
     "size": 5330,
-    "gzipSize": 2214
+    "gzipSize": 2216
   },
   {
     "chunkShortName": "button",
-    "size": 7288,
-    "gzipSize": 2901
+    "size": 7365,
```

Comment:
Kind of strange that this got bigger. Are there some variables or functions shared now?

Reply:
Nothing changed in the button component, the button-tile uses some of the types of the button component and also has the nested button. I am not sure why this could impact the chunk size of the button component 🙈